doc/code/qml_fourier.rst

```diff
@@ -87,10 +87,71 @@ available to a quantum model, the larger the class of functions that model can
 represent, potentially leading to greater utility for quantum machine learning
 applications.
 
+Calculating the frequencies supported by an encoding
```

Comment:
Instead of "encoding" should we instead use something like "circuit", or "quantum model" in the title here? While a key use case is for encoding data, you could in principle compute the coefficients w.r.t. the weights as well, if you mark them as inputs.